build_tools/services.rb

```diff
@@ -177,6 +178,10 @@ def eventstream?(api)
       false
     end
 
+    def support_rbs?(identifier)
+      %w[ lambda ].include?(identifier)
```

Comment:
Define a gem to support RBS.
Is this acceptable?

Reply:
I think we could assume every AWS service gem is supported, so I'd just flat out pass `true` in. We do have use cases (internally) where this generation code is called, and since it defaults to false, that case is fine.

Reply:
I was suggesting we use lambda for reviewing, but I think when we've finalized the code, we may as well launch for all services since it's code generated. :)

Reply:
We *could* consider supporting only a few gems as a "dev preview" then open it up to wider. It depends on our confidence. Thoughts?